src/box/lua/load_cfg.lua

```diff
@@ -497,6 +505,7 @@ local dynamic_cfg = {
     -- do nothing, affects new replicas, which query this value on start
     wal_dir_rescan_delay    = nop,
     wal_cleanup_delay       = private.cfg_set_wal_cleanup_delay,
+    wal_retention_delay     = private.cfg_set_wal_retention_delay,
```

Comment:
Should also be in `dynamic_cfg_skip_at_load`, because you unconditionally pass the delay to wal_init.

Reply:
Done, thank you, didn't know about that.